vllm/_custom_ops.py

```diff
@@ -10,8 +10,8 @@
     logger = init_logger(__name__)
     logger.warning("Failed to import from vllm._C with %r", e)
 
-with contextlib.suppress(ImportError):
-    import vllm._moe_C
+# with contextlib.suppress(ImportError):
+from vllm._moe_C import topk_softmax
```

Comment:
The torch library stuff doesn't allow you to import only parts of a module (afaik).  You'll need to import all of `vllm._moe_C` to get the ops in that library.   And all the moe ops will end up in `torch.ops._moe_C.XXXXX`